README

```diff
@@ -1,12 +1,12 @@
 Percona Server 8.0
-Copyright (c) 2000, 2020, Oracle and/or its affiliates. All rights reserved.
+Copyright (c) 2000, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Minor nit: I would refrain from changing foreign copyright headers, that's up to the copyright owner to change.

Reply:
Reset.